draft-packages/menu/KaizenDraft/Menu/components/MenuListHeading.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react"
+import classnames from "classnames"
+import { Heading } from "@kaizen/typography"
+import { MenuListProps } from "./MenuList"
+import styles from "./MenuListHeading.module.scss"
+
+export type MenuListHeadingProps = Required<Pick<MenuListProps, "heading">> & {
+  listHeadingId: string
+}
+
+export const MenuListHeading = (props: MenuListHeadingProps): JSX.Element => {
+  const { heading, listHeadingId } = props
```

Comment:
Could you apply the `v4()` for the `id` in here to fully encapsulate MenuListHeading's concerns?